AdvancedCalculator/Infrastucture/Calculators/ExpandedCaclulator.cs

```diff
@@ -64,5 +64,22 @@ public bool PerfectNumber(int number)
                 return false;
             
         }
+        public int ArrayMean()
+        {
+            int result = 0;
+            int[] Array1 = new int[10];
+            for (int i = 0; i < 10; i++)
+            {
+                Array1[i] = Convert.ToInt32(Console.ReadLine());
```

Comment:
Nie powinno być to przypisywane w metodzie a wartość powinna być przekazywana przez parametr metody